databases/core.py

```diff
@@ -111,6 +111,10 @@ def __init__(
             async for record in connection.iterate(query):
                 yield record
 
+    async def raw_connection(self) -> typing.Any:
+        async with self.connection() as connection:
+            return await connection.raw_connection()
```

Comment:
This shouldn't be exposed at the top-level. Ie. You *have* to acquire the connection first, in order to access `.raw_connection`

Reply:
That makes sense and sets explicit flow.

Reply:
That also gives clarity. As connection already acquired (and it seems reasonable to avoid spawning the raw_connection from the database level acquiring a new one) we can expose already allocated connection object, that makes it a perfect case for the property.